runtime/moonbase/src/weights/pallet_crowdloan_rewards.rs

```diff
@@ -69,10 +69,10 @@ impl<T: frame_system::Config> pallet_crowdloan_rewards::WeightInfo for WeightInf
 		// Proof Size summary in bytes:
 		//  Measured:  `76347 + x * (550 ±0)`
 		//  Estimated: `66246 + x * (3052 ±5)`
-		// Minimum execution time: 136_696_000 picoseconds.
-		Weight::from_parts(102_379_229, 66246)
```

Comment:
We had to update the `test-crowdloan-democracy.ts` test because of this ref_time increase.

It no-longer allowed the scenario to be performed in a single batch. Now we use 2 transactions.